web/docusaurus.config.js

```diff
@@ -260,6 +260,8 @@ const config = {
         additionalLanguages: ['r'],
       },
     }),
+
+  scripts: [{src: 'https://plausible.io/js/script.js', defer: true, 'data-domain': 'vast.io'}],
```

Comment:
```suggestion
  scripts: [{src: 'https://plausible.io/js/script.js', async:true, defer: true, 'data-domain': 'vast.io'}],
```

https://docusaurus.io/docs/api/docusaurus-config#scripts

> Note that <script> added here are render-blocking, so you might want to add async: true/defer: true to the objects.

This is how plausible themselves do it as well - https://github.com/plausible/docs/blob/08d804dea840f10c49e4118b5ddd5fe0bc722fd8/docusaurus.config.js#L8.

Reply:
Added `async`, thanks.